Global.xcconfig

```diff
@@ -8,11 +8,18 @@ MS_SIM_ARCHS = x86_64 i386
 ARCHS = $(MS_ARM_ARCHS)
 VALID_ARCHS = $(MS_ARM_ARCHS)
 
+// This flag adds -fembed-bitcode-marker in the release configuration
```

Comment:
If you rename the file, you should update environment variable in Bitrise build.